io_export_objex2/interface.py

```diff
@@ -1652,6 +1882,16 @@ def register_interface():
     update_handlers.append(handler_scene_or_depsgraph_update_post_once)
     bpy.app.handlers.load_post.append(handler_load_post)
 
+    # Load Icons
+    # pcoll = bpy.utils.previews.new()
+
+    # my_icons_dir = os.path.join(os.path.dirname(__file__), "icons")
+    # pcoll.load("CYCLE_A", os.path.join(my_icons_dir, "A.png"), 'IMAGE')
```

Comment:
clean this up? I guess it was an attempt that didn't quite yield the results you expected? (yeah custom icons aren't a thing Sadge)

Reply:
They actually worked but I didn't end up using it so I left it there in case if it's ever needed.